src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/additionalConditions/AdditionalCondition.kt

```diff
@@ -0,0 +1,51 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licence.additionalConditions
```

Comment:
<kbd>
<img width="1076" alt="Screenshot 2023-10-27 at 11 48 34" src="https://github.com/ministryofjustice/create-and-vary-a-licence-api/assets/1517745/23904431-ac5a-494c-bdc5-1d86da471b37">
</kbd>